docs/custom-eval.md

```diff
@@ -93,7 +93,7 @@ Generally, most `run` methods will follow the same pattern shown here: loading t
         result = self.completion_fn(prompt=prompt, temperature=0.0, max_tokens=1)
         sampled = result.get_completions()[0]
 
-        evals.record_and_check_match(prompt=prompt, sampled=sampled, expected=sample["answer"])
+        evals.record_and_check_match(prompt=prompt, sampled=sampled, expected=test_sample["answer"])
```

Comment:
This is not strictly needed but less confusing as `sample` here was used outside the loop.